### ISSUE TITLE
Upload .ppt file?

### ISSUE THREAD

**Issue by nicolethenerd** [dphiffer/intro-to-javascript#1]
I love these slides, and I'd love to be able to use them in my own class (with a few of my own additions) - but I don't have Keynote, and .key is a proprietary format (can't be opened by OpenOffice, or PowerPoint).  Any chance you could export your slides as a .ppt file (I understand that this is a feature of Keynote) and upload them to your repo?  Thanks!

**Comment by dphiffer** [comment]
Thanks, I'm glad you find them useful! Have you tried opening up the [PDF version](https://github.com/dphiffer/intro-to-javascript/blob/master/intro-to-javascript.pdf?raw=true)?

**Comment by nicolethenerd** [comment]
Yes, I have, but I can't edit it.  I was hoping to use PowerPoint's annotation feature so I could have your slides on the projector and my own talking points attached to each slide on the screen in front of me.  I figured a ppt version would be useful not just for me, but for anybody else who wants to be able to use/edit your slides in the future.  Thanks!
